client/lib/plans/constants.js

```diff
@@ -208,8 +207,6 @@ export const PLANS_LIST = {
 			isEnabled( 'publicize-scheduling' ) && FEATURE_REPUBLICIZE_SCHEDULING,
 			FEATURE_BUSINESS_ONBOARDING,
 			FEATURE_ADVANCED_SEO,
-			isATEnabledForCurrentSite() && FEATURE_UPLOAD_PLUGINS,
-			isATEnabledForCurrentSite() && FEATURE_UPLOAD_THEMES,
```

Comment:
> This changes behavior slightly so any new user can see this.

I think that's fine because any of those new users could make use of it as well. Tested and works well.